tests/playwright/sanity/modules/floating-buttons/floating-element-page.ts

```diff
@@ -0,0 +1,65 @@
+import WpAdminPage from '../../../pages/wp-admin-page';
+import type { APIRequestContext } from '@playwright/test';
+import EditorPage from '../../../pages/editor-page';
+
+export default class FloatingElementPage extends WpAdminPage {
```

Comment:
Can we remove the inheritance here? It seems to me like 2 separate classes

Reply:
@yotamselementor in my tests I use lots of methods of the base class, it was to avoid using two objects and reusing code. Inheritance make sense to me here, but let me know what you think

Reply:
@nicoladj77 I would rather have these 2 classes as decoupled from each other as possible